Scheduling_Algorithm/C++/First_Come_First_Served.cpp

```diff
@@ -0,0 +1,79 @@
+//First come first serve (FCFS) scheduling algorithm simply schedules the jobs according
+//to their arrival time. The job which comes first in the ready queue will get the
+//CPU first. 
+
+#include<iostream> 
```

Comment:
It would be great to know about terms like `burst_time`, `turn around time` etc.